### PR TITLE
Add vastgoed map to DataPortal via VSD

### PR DESCRIPTION
**Pull request by dolhuis**

### CODE REVIEW

File: src/various_small_datasets/generic/import_data.py [hunk 1]
```diff
@@ -12,6 +15,31 @@
...
+def convert_geojson_srid(geojson, source_srid, target_srid):
+    if isinstance(geojson, dict):
+        if 'coordinates' in geojson:
+            geojson['coordinates'] = convert_coord_arrays(geojson['coordinates'], source_srid, target_srid)
+            return geojson
+        else:
+            return {k: convert_geojson_srid(v, source_srid, target_srid) for k, v in geojson.items()}
+    elif isinstance(geojson, list):
+        return [convert_geojson_srid(i) for i in geojson]
+    return geojson
```
Comment (RRMoelker) [inline src/various_small_datasets/generic/import_data.py]: Seems complicated enough for own file and unit test

File: src/vastgoed/meta/model.json [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    }
+  },
+  "bag_verblijfsobject_geometrie": {
+    "type": "Geo.Point",
+    "properties": {
+      "index": true,
+      "null": true
+    }
+  }
+}
```
Comment (RRMoelker) [inline src/vastgoed/meta/model.json]: Yeah..... Don't agree with writing custom language to build Python models. I get the idea. But creating a specific language is hardly ever a good idea. The problem is that the benefits in configuration are minor compared to the loss in maintainability.

Comment (dolhuis) [reply src/vastgoed/meta/model.json]: Okay. On the other hand it is a direction we are going (see: https://github.com/Amsterdam/schemas and https://github.com/Amsterdam/GOB-Core/blob/master/gobcore/model/gobmodel.json for instance). I think this is not the right place to discuss that, but please join in elsewhere.

File: src/various_small_datasets/generic/import_data.py (hunk 1 above)
Comment (dolhuis) [reply src/various_small_datasets/generic/import_data.py]: Yep, agree.

File: src/vastgoed/meta/model.json (hunk 2 above)
Comment (dolhuis) [reply src/vastgoed/meta/model.json]: Okay. This is a direction we are going in (see for instance https://github.com/Amsterdam/schemas/blob/master/bouwdossiers/bouwdossiers.schema.json and https://github.com/Amsterdam/GOB-Core/blob/master/gobcore/model/gobmodel.json) where the model definition (as single source of truth :-) ) is declared, not built in code. Please join in in the discussions.

**Pull request merged**